devicetypes/smartthings/ecobee-thermostat.src/ecobee-thermostat.groovy

```diff
@@ -32,7 +32,7 @@ metadata {
 		command "switchMode"
 		command "switchFanMode"
 
-		attribute "thermostatSetpoint", "number"
```

Comment:
Since the capability doesn't expose the existence of this attribute, I assume we should be good removing it because no SA should be trying to use it, and if they are they aren't complying with the model.  Right?